test/node/common-safe-require.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const tap = require('tap');
+
+const common = require('../../lib/common');
+
+const existingModule = 'tap';
+const nonExistingModule = 'tp';
```

Comment:
And it might be better to use something more explicitly non-existent like `__not_a_module` here.